pretext/project.py

```diff
@@ -295,13 +297,28 @@ def publish(self,target_name):
         log.info(f"Latest build successfully pushed to GitHub.")
         log.info("(It may take a few seconds for GitHub Pages to reflect any changes.)")
 
-    def xml_source_syntax_is_valid(self,target_name):
+    def xml_source_is_valid(self,target_name):
         target = self.target(target_name)
         return utils.xml_syntax_is_valid(target.source())
 
-    def xml_source_schema_is_valid(self,target_name):
+    def xml_source_validate(self,target_name):
```

Comment:
Maybe `xml_schema_validate` or `xml_schema_is_valid`? I'll never remember the difference between `_is_valid` and `_validate` suffixes.

Reply:
```suggestion
    def xml_schema_validate(self,target_name):
```